contracts/PlayFiLicenseSale.sol

```diff
@@ -180,26 +205,48 @@ IPlayFiLicenseSale
         totalLicenses += amount;
         emit PublicLicensesClaimed(msg.sender, amount, tier, toPay);
     }
-
+    /// @notice Calculates the price, commission and discount for X number of licenses in tier Y given referral code Z
+    /// @param amount The amount of licenses to claim
+    /// @param tier The tier to buy the licenses from
+    /// @param referral A referral code that can give discounts.
+    /// @return toPay The amount of ETH that should be paid by the claimer.
+    /// @return commission The commission in ETH that the referrer will get.
+    /// @return discount The discount in ETH the claimer will get
     function paymentDetailsForReferral(uint256 amount, uint256 tier, string calldata referral) public view returns (uint256 toPay, uint256 commission, uint256 discount) {
         uint256 fullPrice = tiers[tier].price * amount;
         discount = fullPrice * referrals[referral].discountPercentage / 100;
         commission = fullPrice * referrals[referral].commissionPercentage / 100;
         toPay = fullPrice - discount;
     }
 
+    /// @notice Returns tier details provided a tier id
+    /// @param id The tier id
+    /// @return tier The tier
     function getTier(uint256 id) public view returns(Tier memory tier) {
         tier = tiers[id];
     }
 
+    /// @notice Returns referral details provided a referral id
+    /// @param id The referral id
+    /// @return referral The referral
     function getReferral(string memory id) public view returns(Referral memory referral) {
         referral = referrals[id];
     }
 
+    /// @notice Sets referral details
+    /// @param code The referral code to be used when claiming
+    /// @param receiver The receiver address for the commissions
+    /// @param commission The percentage of the total price to be used as a commission
+    /// @param discount The percentage of the total price to be used as a discount
     function setReferral(string memory code, address receiver, uint256 commission, uint256 discount) public onlyReferralManager {
         _setReferral(code, receiver, commission, discount);
     }
 
+    /// @notice Sets tier details
+    /// @param ids the ids of the tiers to set
+    /// @param prices the prices of each tier to set
+    /// @param individualCaps the maximum amount of licenses that can be claimed per address for the tiers.
+    /// @param totalCaps the maximum amount of licenses that can be claimed in total for the tiers.
```

Comment:
super nit (feel free to ignore): the explanation following the parameter is previously uppercased. Previous comments looked like so: param ids **T**he ids of the tiers to set